### ISSUE TITLE
[Fatal Error] Can not open table with JSON field

### ISSUE THREAD

**Issue by qts-mostafa** [phpmyadmin/phpmyadmin#12132]
### Steps to reproduce
1. Create a table with one INT auto increment field and one JSON field
2. Open the table (The same thing with or without data)

### Expected behaviour
I should be able to view and edit the results and all javascript functions show work normally.

### Actual behaviour
I get the following error and everything in the page freezes
```
Notice in ./libraries/dbi/DBIMysqli.php#515
 Undefined offset: 245

Backtrace

./libraries/DatabaseInterface.php#2637: PMA\libraries\dbi\DBIMysqli->getFieldsMeta(object)
./libraries/sql.lib.php#1845: PMA\libraries\DatabaseInterface->getFieldsMeta(object)
./libraries/sql.lib.php#2185: PMA_getQueryResponseForResultsReturned(
object,
array,
string 'tst',
string 'aa',
NULL,
NULL,
object,
string './themes/pmahomme/img/',
string '1',
integer 1,
NULL,
NULL,
NULL,
NULL,
NULL,
string 'SELECT * FROM `aa`',
NULL,
)
./libraries/sql.lib.php#2065: PMA_executeQueryAndGetQueryResponse(
array,
boolean true,
string 'tst',
string 'aa',
NULL,
NULL,
NULL,
NULL,
NULL,
NULL,
string '',
string './themes/pmahomme/img/',
NULL,
NULL,
NULL,
string 'SELECT * FROM `aa`',
NULL,
NULL,
)
./sql.php#218: PMA_executeQueryAndSendQueryResponse(
array,
boolean true,
string 'tst',
string 'aa',
NULL,
NULL,
NULL,
NULL,
NULL,
NULL,
string '',
string './themes/pmahomme/img/',
NULL,
NULL,
NULL,
string 'SELECT * FROM `aa`',
NULL,
NULL,
)
```
Where "aa" is my table name, and "tst" is my database name.

I tried to refresh the page but I found another issue appears. Whatever the page I try to access it just shows the fatal error below with white background.
```Fatal error: Cannot redeclare __() (previously declared in /usr/share/phpmyadmin/libraries/sql-parser/src/common.php:17) in /usr/share/phpmyadmin/libraries/php-gettext/gettext.inc on line 290```

It does not open any page till I remove the cookies. I also noticed this javascript error before refreshing
```
Uncaught SyntaxError: Unexpected identifier
http://192.168.1.251/phpmyadmin/js/get_scripts.js.php?scripts%5B%5D=jquery/jquery.uitablefilter.js&scripts%5B%5D=gis_data_editor.js&scripts%5B%5D=multi_column_sort.js&call_done=1&v=4.6.0 Invalid JSON!Unexpected token F
```
```
http://192.168.1.251/phpmyadmin/sql.php?db=tst&token=2dcc80425923ee3f8b32e53d065ff834&goto=db_structure.php&table=aa&pos=0
```
I think it is because the previous error. It can not parse the response "Fatal error" as javascript.

### Server configuration
**Operating system**:
Centos 6 64bit

**Web server:**
Apache/2.2.15

**Database:**
5.7.11 - MySQL Community Server (GPL)

**PHP version:**
5.6.18

**phpMyAdmin version:**
4.6.0

### Client configuration
**Browser:**
Google chrome: Version 49.0.2623.75 (64-bit)

**Operating system:**
Ubuntu 14.04 64bit

**Comment by ibennetch** [comment]
Exists in current master with sql-parser `@dev` as well.

**Comment by ibennetch** [comment]
The error message exists in current master with sql-parser `@dev` as well.

However, if I click "Ignore" to the popup warning window, I'm taken to the Browse page as expected and I'm able to work with the table without trouble. I have not been able to reproduce the fatal error.

**Comment by qts-mostafa** [comment]
**Once opened the table**
![screenshot from 2016-03-27 13 51 33](https://cloud.githubusercontent.com/assets/9843910/14065273/8e04503c-f423-11e5-9c2a-f738c601fba6.png)

**After refresh**
![screenshot from 2016-03-27 13 48 31](https://cloud.githubusercontent.com/assets/9843910/14065271/8dc37fbc-f423-11e5-9639-463b8864b1a3.png)

**Comment by ibennetch** [comment]
Testing again with 4.6.0 I also get the fatal error.

**Comment by ibennetch** [comment]
This has been fixed with #12141 and will be a part of phpMyAdmin 4.6.1.

You can test the fix yourself by downloading the snapshot from https://github.com/phpmyadmin/phpmyadmin/archive/QA_4_6.zip
